lib/GADS/Helper/ConditionBuilder.pm

```diff
@@ -0,0 +1,105 @@
+use utf8;
+
+package GADS::Helper::ConditionBuilder;
```

Comment:
I like the use of this class in principle. However, I think we should also integrate it into `GADS::Records::_search_construct` so as to create common code.

Reply:
This is to be implemented

Reply:
I will take a look at this